content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -171,47 +136,35 @@ if (msgFromClient.equalsIgnoreCase("BYE"))
     }  
 }
 ```
-
 The output of this  program is:
-
 ```
 Connected
 Client: Hey there 
-Client: am doing this for the first time
+Client: I am doing this for the first time
 Client: BYE
 Closing connection
 ```
-
 ### Client-side Programming
 #### Establish a Socket Connection
-
 A Socket instance can be obtained in two ways: 
     
-1. Server side receive it as a return value of the accept method.
-2. You can instatiate it as a Socket like so:
-
+1. Server side receives it as a return value of the accept method.
+2. You can instantiate it as a Socket like so:
 ```java
 socket = new Socket("localhost", 5000);
 ```
-
-localhost - is a domain name like google.com,which takes you to your computer. It is resolved to IP address 127.0.0.1 .
+localhost - is a domain name like google.com, which takes you to your computer. It is resolved to IP address 127.0.0.1.
 Additionally, the second argument is a port number.
-
 Like before we create I/O streams via the socket object:
-
 ```java
 inputStreamReader = new InputStreamReader(socket.getInputStream());  
 outputStreamWriter = new OutputStreamWriter(socket.getOutputStream());
 ``` 
-
 Client-side, we need to use a Scanner object to get input from the user from the console/terminal. 
-
 ```java
 Scanner scanner = new Scanner(System.in);
 ```
-
-System.in specifies that we are getting the keyboard input.
-
+`System.in` specifies that we are getting the keyboard input.
 #### Methods used in Client Socket
 1. `public InputStream getInputStream()` - returns an InputStream object for receiving data. Our Server will keep on receiving messages until client sends "BYE".
```

Comment:
> 1. `public InputStream getInputStream()` - returns an InputStream object for receiving data. Our Server will keep on receiving messages until client sends "BYE".
> 2. `public OutputStream getOutputStream()`- returns an OutputStream for sending data.

Change the numbers in this list to dashes like so:

> \- `public InputStream getInputStream()` - returns an InputStream object for receiving data. Our Server will keep on receiving messages until client sends "BYE".
> \- `public OutputStream getOutputStream()`- returns an OutputStream for sending data.

When you listed the methods in the server socket, you listed them in an unordered list (dashes instead of numbers). You should keep it consistent and do the same here

Reply:
I think I missed this last time, sorry if that is the case.